Dockerfile

```diff
@@ -98,6 +98,9 @@ RUN apk add --update                                    \
     chown -R $JIRA_USER:$JIRA_GROUP ${JIRA_INSTALL} &&  \
     chown -R $JIRA_USER:$JIRA_GROUP ${JIRA_SCRIPTS} &&  \
     chown -R $JIRA_USER:$JIRA_GROUP /home/${JIRA_USER} &&  \
+    # add helpers
+    wget -O /usr/local/bin/wait-for-it --no-check-certificate http://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh && \
```

Comment:
Please remove the option --no-check-certificate. It's insecure and I am already accessing github with my Dockerfile so it should work with ssl.